R/range-selector.R

```diff
@@ -56,7 +56,11 @@ dyRangeSelector <- function(dygraph,
   if (!is.null(dateWindow)) {
     if (length(dateWindow) != 2)
       stop("dateWindow must be vector of length 2 that is convertible to POSIXct")
-    selector$dateWindow <- sapply(USE.NAMES = FALSE, dateWindow, asISO8601Time)
+    # TODO: Rename dateWindow argument?
```

Comment:
An alternative would be to use `dyRangeSelector <- function(dygraph, window, height, fillColor, strokeColor, keepMouseZoom, retainWindow, dateWindow, retainDateWindow)`.
But I'm not sure if this makes things more easy or complicated to understand if you begin working with dygraphs